packages/core-snapshots/src/manager.ts

```diff
@@ -49,7 +49,7 @@ export class SnapshotManager {
 
         if (params.truncate) {
             await this.database.truncate();
-            params.lastBlock = undefined;
+            params.lastBlock = null;
```

Comment:
- https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined
- https://basarat.gitbooks.io/typescript/docs/javascript/null-undefined.html

Reply:
Changed back to `undefined` in d096d66f1 to silence the tslint warning.

Notice however that the second link says

> Something hasn't been initialized : undefined
> Something is currently unavailable: null

and in this case the last block is not available because there are no blocks, so `null` should be used. This is also in line with the `getLastBlock()` which returns `null` if there are no blocks.